app/src/main/res/layout/fragment_add_bookmark_folder.xml

```diff
@@ -14,7 +14,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
-        android:text="@string/bookmark_name_label"
+        android:layout_marginLeft="4dp"
+        android:text="NAME"
```

Comment:
Do we need to hardcode the string here!  I guess we must use the resource @string/bookmark_name_label